### PR TITLE
XRDDEV-403 technical log

### PR DESCRIPTION
**Pull request by jansu76**
https://jira.niis.org/browse/XRDDEV-403

Updates to proxy-ui-api technical logging to better follow common conventions and OWASP recommendations. Some documentation updates.

### CODE REVIEW

File: doc/Manuals/ug-ss_x-road_6_security_server_user_guide.md [hunk 1]
```diff
@@ -1884,6 +1884,7 @@ The most important system services of a security server are as follows.
  `xroad-jetty `     | Application server running the user interface | `/var/log/xroad/jetty/`
  `xroad-proxy`      | Message exchanger                             | `/var/log/xroad/proxy.log`
  `xroad-signer`     | Manager process for key settings              | `/var/log/xroad/signer.log`
+ `xroad-proxy-ui-api`  | Management UI and REST API                 | `/var/log/xroad/proxy_ui_api.log` and <br/>`/var/log/xroad/proxy_ui_api_access.log`
```
Comment (iluwatar) [inline doc/Manuals/ug-ss_x-road_6_security_server_user_guide.md:1887]: The version number of the doc should be incremented too

File: src/proxy-ui-api/src/main/resources/common-application.yml [hunk 2]
```diff
@@ -70,7 +70,7 @@ file-upload-endpoints:
       pathEnding: /tls-certificates
     -
       httpMethod: POST
-      pathEnding: /certificates
+      pathEnding: /token-certificates
```
Comment (iluwatar) [inline src/proxy-ui-api/src/main/resources/common-application.yml:73]: This change seems unrelated to logging, intentional?

Comment (jansu76) [reply src/proxy-ui-api/src/main/resources/common-application.yml:73]: Yes it is intentional, I noticed that endpoint names had been updated `/certificates` -> `/token-certificates`, but this configuration file had not been updated accordingly. It is unrelated to logging, indeed.

File: doc/Manuals/ug-ss_x-road_6_security_server_user_guide.md (hunk 1 above)
Comment (jansu76) [reply doc/Manuals/ug-ss_x-road_6_security_server_user_guide.md:1888]: fixed
